packages/tasks_panel/src/commands/edit-task.ts

```diff
@@ -4,16 +4,31 @@ import { TaskTreeItem } from "../view/task-tree-item";
 import { messages } from "../i18n/messages";
 import { getSWA } from "../utils/swa";
 import { createTaskEditorPanel, disposeTaskSelectionPanel, getTaskInProcess } from "../panels/panels-handler";
+import { ITasksProvider } from "../services/definitions";
+import { find, has } from "lodash";
+import { serializeTask } from "../utils/task-serializer";
+import { getLogger } from "../logger/logger-wrapper";
 
 export async function editTreeItemTask(
+  tasksProvider: ITasksProvider,
```

Comment:
Is it OK to change this API? it is not exposed/being used by others?

Reply:
sure.